backend/app/src/main/java/com/moh/phlat/backend/esb/json/MaintainFacilityResponse.java

```diff
@@ -26,82 +25,80 @@ public class MaintainFacilityResponse implements PlrResponse {
 	private static final String DUPLICATE_FACILITY_RESPONSE_CODE = "PRS.SYS.ADR.UNK.1.0.7055";
 	private static final String FACILITY_LOADED_RESPONSE_CODE = "PRS.PRV.OID.CRE.0.0.0003";
 
-	private ProcessData data;
+	private static ObjectMapper objectMapper;
+	static {
+		JSON_DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
+		
+		objectMapper = new ObjectMapper();
+		objectMapper.setDateFormat(JSON_DATE_FORMAT);
+		objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+		objectMapper.disable(SerializationFeature.WRITE_DATES_WITH_ZONE_ID);
+		objectMapper.disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE);
+		objectMapper.setSerializationInclusion(Include.NON_NULL);
+	}
 	
-	@Getter
-	private String facilityId;
+	private ProcessData data;
```

Comment:
```suggestion
	private ProcessData processData;
```
Please make this change everywhere